Sources/SwiftFusion/Inference/MonteCarloEM.swift

```diff
@@ -93,10 +93,12 @@ public struct MonteCarloEM<ModelType: McEmModel> {
   
   /// Run Monte Carlo EM given unlabeled data
   public mutating func run(with data:[ModelType.Datum],
+                           modelInitializer: (_ data: [ModelType.Datum], _ entropySource: inout AnyRandomNumberGenerator) -> ModelType,
```

Comment:
As I said, I don't like this change at all. The inits were prescribed by the Protocol. This makes a weird in-between design which is neither here nor there.